### ISSUE TITLE
Exporting the operator broadcast_tensors to ONNX opset version 12 is not supported

### ISSUE THREAD

**Issue by anubane** [pytorch/pytorch#69073]
## 🐛 Bug

On attempting to convert a custom model to ONNX with opset version 12, I receive the error message:
```bash
Exporting the operator broadcast_tensors to ONNX opset version 12 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub.
```

## Expected behavior

The model must successfully be converted to ONNX.

## Environment

 - PyTorch Version (e.g., 1.0): **1.9.1**
 - OS (e.g., Linux): **Ubuntu 18.04.6 LTS**
 - How you installed PyTorch (`conda`, `pip`, source): **pip**
 - Python version: **3.6.9**
 - CUDA/cuDNN version: **11.5/10.0**
 - GPU models and configuration: **Tesla V100**

## Additional Details

```python
torch.onnx.export(model,
                  input_tuple,
                  'model.onnx',
                  verbose=True,
                  input_names=input_names,
                  output_names=output_names,
                  export_params=True,
                  do_constant_folding=False,
                  opset_version=12,
                  training=torch.onnx.TrainingMode.TRAINING,
                  dynamic_axes={'Input1' : {0 : 'batch_size'}, 'Input2' : {0 : 'batch_size'}})
```

The base model available at https://github.com/mlcommons/training/tree/d0a86d67e41186835cf3f484f7e3ef00d02b820a/single_stage_detector

was modified to incorporate loss as part of the model. Further, the model is being exported in torch.onnx.TrainingMode.TRAINING mode.

**Comment by BowenBao** [comment]
hi @anubane could you check with latest pytorch nightly? There was a PR merged in master that adds support for `broadcast_tensors` #61560

**Comment by garymm** [comment]
Closing since we think this is done and no response from @anubane.
